### DISCUSSION TITLE
Sending alerts to multiple slack channels for multiple groups

### DISCUSSION THREAD

**Discussion by mani9223-oss** [TwiN/gatus#276] [Q&A] [answered]
I would like to send alerts from multiple groups to multiple slack channels.

@TwiN Is this possible?

**Comment by TwiN** [comment, marked as answer]
The Slack alerting provider doesn't currently have overrides implemented, but it should be fairly simple to implement if you'd like throw a PR together that I can review :) 

There's an example at https://github.com/TwiN/gatus/pull/271
